lib/internal/Magento/Framework/App/Scope/Validator.php

```diff
@@ -83,7 +83,7 @@ private function validateScopeCode($scopeCode)
             throw new LocalizedException(new Phrase('A scope code is missing. Enter a code and try again.'));
         }
 
-        if (!preg_match('/^[a-z]+[a-z0-9_]*$/', $scopeCode)) {
+        if (!preg_match('/^[a-z]+[a-z0-9_]*$/i', $scopeCode)) {
             throw new LocalizedException(new Phrase(
                 'The scope code can include only lowercase letters (a-z), numbers (0-9) and underscores (_). '
```

Comment:
Please correct the exception message as well (remove `lowercase` from it) as it now accesses both lowercase and uppercase letters.